Extract data from URLs/mod.ts

```diff
@@ -0,0 +1,153 @@
+import { parse } from 'npm:tldts'
```

Comment:
I think you don't need this here, if you use the plain [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) web api, it will return all the data you need in roughly the same time.

Reply:
Everything here is just me commenting thinking you were adding this code